Sources/Kitura/FileResourceServer.swift

```diff
@@ -22,6 +22,11 @@ class FileResourceServer {
     /// if file found - send it in response
     func sendIfFound(resource: String, usingResponse response: RouterResponse) {
         guard let resourceFileName = getFilePath(for: resource) else {
+            do {
+                try response.status(.notFound).end()
+            } catch {
+                Log.error("failed to send not found response")
+            }
```

Comment:
As per the [w3 spec](https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.4) - "Except when responding to a HEAD request, the server SHOULD include an entity containing an explanation of the error situation, and whether it is a temporary or permanent condition."

We do a similar thing (sending an error message for the body) [here](https://github.com/IBM-Swift/Kitura/blob/master/Sources/Kitura/Router.swift#L342-L343)